esphome/components/neewerlight_ct/neewer_state_output.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "esphome/components/output/float_output.h"
+
+#ifdef USE_ESP32
+
+namespace esphome {
+namespace neewerlight_ct {
```

Comment:
```suggestion
namespace esphome::neewerlight_ct {
```
Please switch all of them to use the modern style

Reply:
done